### DISCUSSION TITLE
Updated user list?

### DISCUSSION THREAD

**Discussion by ryaneford** [infinite-wait/blockout2024#11] [General]
Just wanted to know how you're getting and maintaining the list for your script.  :)

**Comment by incandenza-11** [comment]
Hey @ryaneford! The list was updated yesterday and it now has +900 usernames, but if you find that there's something missing feel free to contribute to the project :)

**Reply by ryaneford** [reply]
I follow a couple of places that seem to have some updated lists. How are you exporting the usernames or quickly cleaning up the data to put into plain text?

**Reply by molinto** [reply]
Where are these lists please?

**Comment by ryaneford** [comment]
Any update?
